.gitignore

```diff
@@ -33,3 +33,6 @@ psalm/psalm-baseline-pr.xml
 config/bash.env
 .vscode
 phpcs.xml
+
+# Language binaries are created during the docker build process or using 'make compilelang'
+*.mo
```

Comment:
Ideally in another PR but fine for now.

Reply:
Moved to GH-289 (pending rebase here)

Reply:
Merged, so this PR can be rebased, although I still hope that there will be a day where that will no longer be necessary.